bemani/api/objects/catalog.py

```diff
@@ -165,11 +165,8 @@ def __format_extras(self) -> Dict[str, List[Dict[str, Any]]]:
 
     @property
     def music_version(self) -> int:
-        if self.game == GameConstants.IIDX:
```

Comment:
I think we should change to `if self.game in {GameConstants.IIDX, GameConstants.MUSECA]` because otherwise we end up requesting and sending empty lists for other games that have no omni support if omnimix is requested. The current setup sends the base game's scores/etc which, while incomplete, is still better than nothing.

Reply:
I plan to extend omnimix to popn, jubeat(already done on my main repository), and maybe even reflec beat. Should I just add every individual case as I add omnimix support? I feel like that just gets a bit overbearing but if it's preferred to explicitly say that there is no omnimix support for a game then I'll change it.

Reply:
I prefer explicit, because it forces you to "turn on" support for omnimix in games. The reason is such: Say a remote server does support omnimix and this server doesn't. If we turn on omnimix support for all games, when a game requests scores, we will return an empty list. If we explicitly turn on omnimix per game, then we would at least return scores for the base game an omnimix is based off of.